section2/exercises/if-statements.js

```diff
@@ -31,19 +31,26 @@ if (weather === 'sunny') {
   console.log('good to go!');
 }
 
-/* 
+/*
 YOU DO:
-Using the dogAge variable defined below, 
-determine if a dog is a puppy (2 or younger), 
+Using the dogAge variable defined below,
+determine if a dog is a puppy (2 or younger),
 an adult, or elderly (10 or older).
 
 Log to the console the appropriate age range
 (puppy, adult, elderly).
 */
 
 var dogAge = 3;
+if(dogAge <= 2){
```

Comment:
Beautiful solution! Just make sure to clean up your use of spaces - ex. pad the `()` that wrap a condition with a space on each side. 

```
if (dogAge <= 2) {
```